### PR TITLE
[engine] Do not allow to overwrite default composite paths

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: docs/modules/ROOT/pages/tests-configuration.adoc [hunk 1]
```diff
@@ -107,6 +107,10 @@ NOTE: The properties marked with *bold* are mandatory.
 |`report.text-length-diff-threshold`
 |`100`
 |Defines text threshold length to enable unified diff report publishing on failure, for variable comparisons. If you want to have this report despite the size of the compared data set 0.
+
+|`engine.composite-paths`
```
Comment (uarlouski) [inline docs/modules/ROOT/pages/tests-configuration.adoc:111]: isn't engine too technical for end users?

**Review by uarlouski — APPROVED** [review]

File: docs/modules/ROOT/pages/tests-configuration.adoc [hunk 2]
```diff
@@ -726,6 +730,43 @@ http.auth.password=ENC(WVnXhzG704KRZZFYL7hfpA4YoSm8V2F5eNXyr7CRYCichteHvNS3nB23z
 
 Please see <<_externalized_configuration>> to get more information about how encryptor password can be passed to the tests.
 
+== Composite steps
+
+https://jbehave.org/reference/stable/composite-steps.html[Composite steps] allows to reuse set of actions via step with a custom wording.
```
Comment (valfirst) [inline docs/modules/ROOT/pages/tests-configuration.adoc:735]:
```suggestion
https://jbehave.org/reference/stable/composite-steps.html[Composite steps] allow to reuse set of actions via custom steps with own wordings.
```

File: docs/modules/ROOT/pages/tests-configuration.adoc [hunk 3]
```diff
@@ -726,6 +730,43 @@ http.auth.password=ENC(WVnXhzG704KRZZFYL7hfpA4YoSm8V2F5eNXyr7CRYCichteHvNS3nB23z
...
+
+=== How to use composite steps
+
+. Define composite step:
```
Comment (valfirst) [inline docs/modules/ROOT/pages/tests-configuration.adoc:739]:
```suggestion
. Define composite step in `*.steps` file, e.g. `steps/ui/my-app.steps`:
```

File: docs/modules/ROOT/pages/tests-configuration.adoc [hunk 4]
```diff
@@ -726,6 +730,43 @@ http.auth.password=ENC(WVnXhzG704KRZZFYL7hfpA4YoSm8V2F5eNXyr7CRYCichteHvNS3nB23z
...
++
+.Compsite step definition
+[source,gherkin]
+----
+Composite: When I login with username `$username` and password `$password`
+When I enter `<username>` in field located `id(username)`
+When I enter `<password>` in field located `id(password)`
+When I click on element located `id(signInButton)`
```
Comment (valfirst) [inline docs/modules/ROOT/pages/tests-configuration.adoc:747]:
```suggestion
When I enter `<username>` in field located by `id(username)`
When I enter `<password>` in field located by `id(password)`
When I click on element located by `id(signInButton)`
```

File: docs/modules/ROOT/pages/tests-configuration.adoc [hunk 5]
```diff
@@ -726,6 +730,43 @@ http.auth.password=ENC(WVnXhzG704KRZZFYL7hfpA4YoSm8V2F5eNXyr7CRYCichteHvNS3nB23z
...
+----
+
+. Configure composite paths
```
Comment (valfirst) [inline docs/modules/ROOT/pages/tests-configuration.adoc:750]:
```suggestion
. Configure paths to files with custom steps
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

**Pull request merged**